merchantapi/src/main/java/fi/hel/verkkokauppa/configuration/service/NamespaceService.java

```diff
@@ -163,7 +163,7 @@ private NamespaceModel getNamespaceModelByNamespace(String namespace) {
      * @param key       The key of the configuration value you want to retrieve.
      * @return A String of configuration value or null
      */
-    public String getConfigurationValueByMerchantIdAndNamespaceAndKey(String namespace, String key) {
+    public String getConfigurationValueByNamespaceAndKey(String namespace, String key) {
```

Comment:
Renamed this, because i noticed it does only fetch the ConfigurationValue for the namespace specifically.